ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/subscription/logs/LogsSubscriptionServiceTest.java

```diff
@@ -90,8 +90,9 @@ public void singleMatchingLogEvent() {
     final List<TransactionReceipt> receipts = blockWithReceipts.getReceipts();
 
     final int txIndex = 1;
-    final int logIndex = 1;
-    final Log targetLog = receipts.get(txIndex).getLogsList().get(logIndex);
+    final int logListIndex = 1;
```

Comment:
To make the code more readble I suggest the following renames:

1. `logListIndex` -> `logIndexInTransaction`
2. `logIndex` -> `logIndexInBlock`